release/terraform/main.tf

```diff
@@ -0,0 +1,42 @@
+terraform {
+  backend "gcs" {
+    prefix  = "terraform/state"
+  }
+}
+
+provider "google" {
+  project = var.project
+}
+
+resource "google_storage_bucket" "regional_bucket" {
+  for_each                    = toset(var.bq_regions)
+  name                        = "${var.project}-lib-${each.value}"
+  uniform_bucket_level_access = true
+  location                    = each.key
+  force_destroy               = false
+}
```

Comment:
Add access block which sets the buckets as publicly readable to allAuthenticedUsers but only for bqutil project